windows-installer/construct.py

```diff
@@ -70,13 +70,39 @@ def _copy_assets(build_path, repo_path):
 def _generate_pynsist_config(repo_path, build_path):
     print('Generate pynsist configuration')
 
+    pywin32_paths_file = os.path.join(build_path, 'pywin32_paths.py')
+
+    # Pywin32 uses non-standard folders to hold its packages. We need to instruct pynsist bootstrap
+    # explicitly to add them into sys.path. This is done with a custom "pywin32_paths.py" that is
+    # referred in the pynsist configuration as an "extra_preamble".
+    # Reference example: https://github.com/takluyver/pynsist/tree/master/examples/pywebview
+    with open(pywin32_paths_file, 'w') as file_h:
+        file_h.write('''\
+pkgdir = os.path.join(os.path.dirname(installdir), 'pkgs')
+
+sys.path.extend([
+    os.path.join(pkgdir, 'win32'),
+    os.path.join(pkgdir, 'win32', 'lib'),
+])
```

Comment:
What made you remove `Pythonwin` here which is present at https://github.com/takluyver/pynsist/blob/master/examples/pywebview/pywin32_paths.py#L4

Reply:
Because it is related to Windows UI utilities, and we do not use that since Certbot is a pure CLI project. All the integrations tests are passing without it, by the way.

Reply:
Thanks for the explanation. Just trying to understand the change.

As an aside, did you know that the first 10 or so Certbot releases had and defaulted to a TUI: https://www.youtube.com/watch?v=Ya6t8nMclos